src/main/java/nl/tudelft/ewi/devhub/server/web/resources/repository/AbstractProjectResource.java

```diff
@@ -188,8 +192,8 @@ protected AbstractProjectResource(final TemplateEngine templateEngine,
 			} else throw e;
 		}
 
-		if (branchDeletion != null) {
-			parameters.put("deleteSuccessful", branchDeletion);
+		if (branchDeletionStatus != null) {
+			parameters.put("deleteStatus", branchDeletionStatus);
```

Comment:
I more than 2 values to specify the type of alert that is shown. It is the easiest and simplest way of implementing a confirmation for deleting ahead branches